src/components/CohortBuilder/Summary/Cards/SummaryCard.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement, useRef, RefObject } from 'react';
+import { Card as AntdCard } from 'antd';
+import './SummaryCard.css';
+
+type CardProps = {
+  title?: string;
+  loading?: boolean;
+  children: ReactElement[] | any | Function;
```

Comment:
or any?

Reply:
I put it in the middle so nobody can see it! 🤣 

Joke aside, I'll fix that for sure!